.github/workflows/integration-tests.yml

```diff
@@ -10,12 +10,15 @@ jobs:
   integration-test:
     strategy:
       fail-fast: true
+      matrix:
+        exasol_version: ["7.1.17", "8.18.1"]
```

Comment:
```suggestion
        exasol_version: ["7.1.21", "8.18.1"]
```

Reply:
ITDE only supports some versions, and 7.1.17 is the latest supported 7.1.x.